packages/backend/src/persistence/storage/postgres/getMessages.ts

```diff
@@ -1,17 +1,21 @@
 import { PrismaClient } from '@prisma/client';
-import { getOrCreateAccount } from './utils/getOrCreateAccount';
-
-const PAGE_SIZE = 100;
 
 export const getMessages =
     (db: PrismaClient) =>
-    async (ensName: string, encryptedContactName: string, page: number) => {
+    async (
+        ensName: string,
+        encryptedContactName: string,
+        size: number,
```

Comment:
I think `chunkSize`/`pageSize` and `chunkOffset`/`pageOffset` or something similar would be clearer. I hear `size`, I think kilobytes. I hear `offset`, I think data structure or something.